src/k8s/pkg/component/component_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"path/filepath"
 	"testing"
 
+	mocks "github.com/canonical/k8s/pkg/component/mock"
```

Comment:
Any reason to rename from `mock`? 
```suggestion
	"github.com/canonical/k8s/pkg/component/mock"
```

Reply:
I'm also importing the snap mock so go gets confused since both the snap mock and the initializer mock get imported. Do you have a better idea?

Reply:
I would recommend `snapmock` and `componentmock`

Reply:
Good call! Thanks